config/manager/manager.yaml

```diff
@@ -1,66 +1,37 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  labels:
-    controller-tools.k8s.io: "1.0"
-  name: system
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: controller-manager-service
-  namespace: system
   labels:
     control-plane: controller-manager
-    controller-tools.k8s.io: "1.0"
-spec:
-  selector:
-    control-plane: controller-manager
-    controller-tools.k8s.io: "1.0"
-  ports:
-  - port: 443
+  name: system
 ---
 apiVersion: apps/v1
-kind: StatefulSet
+kind: Deployment
 metadata:
   name: controller-manager
   namespace: system
   labels:
     control-plane: controller-manager
-    controller-tools.k8s.io: "1.0"
 spec:
   selector:
     matchLabels:
       control-plane: controller-manager
-      controller-tools.k8s.io: "1.0"
-  serviceName: controller-manager-service
+  replicas: 1
   template:
     metadata:
       labels:
         control-plane: controller-manager
```

Comment:
Do we need to prefix this? @detiber

Reply:
Yes, we should, but maybe that is something we should do through a kustomize patch, though.

Reply:
Added!